test/gtest/uct/ib/test_ib.cc

```diff
@@ -534,14 +546,15 @@ class test_uct_event_ib : public test_uct_ib {
 protected:
     static const unsigned EVENTS = UCT_EVENT_RECV | UCT_EVENT_SEND_COMP;
 
-    struct pollfd wakeup_fd;
     size_t length;
     uint64_t test_ib_hdr;
     mapped_buffer *m_buf1, *m_buf2;
     static size_t bcopy_pack_count;
+    static uct_test::async_event_ctx m_async_event_ctx;
```

Comment:
can we avoid this global variable?

Reply:
unfortunately, no
`async_event_handler()` is a static method that we pass to UCT

Reply:
UPD.: will pass pointer to the class